docs/format/db.rst

```diff
@@ -58,6 +71,29 @@ Related tools:
    * Ex: CLB is solved by first solving LUT bits, and then solving FF bits
 
 
+Interconnect PIP Tags
+^^^^^^^^^^^^^^^^^^^^^
+
+Tags for interconnect PIPs are stored in the `segbits_int_l.db` and `segbits_int_r.db` database files. For example, look at `segbits_int_l.db` for the bits that configure the `INT_L` tile in a `CLBLL_L` or `CLBLM_L` segment.
+
+Tags that enable interconnect PIPs have the following syntax: `<tile_type>.<destination_wire>.<source_wire>`.
+
+The `<tile_type>` may be `INT_L` or `INT_R`. The destination and source wires are wire names in that tile type. For example, consider the following entry in `segbits_int_l.db`: `INT_L.NL1BEG1.NN6END2 07_32 12_33`
+
+This means that the bits `07_32` and `12_33` must be set in the segment to drive the value from the wire `NN6END2` to the wire `NL1BEG1`.
+
+CLB Configurations Tags
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Tags for CLB tiles use a dot-separated hierarchy for their tag names. For example the tag `CLBLL_L.SLICEL_X0.ALUT.INIT[00]` documents the bit position of the LSB LUT init bit for the ALUT for the slice with even X coordinate within a `CLBLL_L` tile. (There are 4 LUTs in a slice: ALUT, BLUT, CLUT, and DLUT. And there are two slices in a CLB tile: One with an even X coordinate using the `SLICEL_X0` namespace for tags, and one with an odd X coordinate using the `SLICEL_X1` namespace for tags.)
```

Comment:
I suggest monospace (double backticks) instead of single backticks for inline examples.